<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShareDlg</class>
 <widget class="QDialog" name="ShareDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>95</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>请选择要导出的共享目录：</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="sharePath_tableWidget">
   <property name="geometry">
    <rect>
     <x>655</x>
     <y>170</y>
     <width>691</width>
     <height>221</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>150</number>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>路径</string>
    </property>
   </column>
  </widget>
  <widget class="SourcefileWidget" name="sourcefileWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>615</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sourcePath_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>e:/</string>
   </property>
  </widget>
  <widget class="SourcefileWidget" name="targetfileWidget" native="true">
   <property name="geometry">
    <rect>
     <x>651</x>
     <y>390</y>
     <width>700</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>源目录：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>923</x>
     <y>130</y>
     <width>423</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="OutputShareBtn">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>导出</string>
      </property>
      <property name="icon">
       <iconset resource="fastvideo.qrc">
        <normaloff>:/images/copyout.png</normaloff>:/images/copyout.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="AddSharePathBtn">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>共享目录</string>
      </property>
      <property name="icon">
       <iconset resource="fastvideo.qrc">
        <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="ModifySharePathBtn">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>修改</string>
      </property>
      <property name="icon">
       <iconset resource="fastvideo.qrc">
        <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="DelSharePathBtn">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
      <property name="icon">
       <iconset resource="fastvideo.qrc">
        <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SourcefileWidget</class>
   <extends>QWidget</extends>
   <header>sourcefilewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="fastvideo.qrc"/>
 </resources>
 <connections/>
</ui>
